xlive/H2MOD.cpp

```diff
@@ -696,10 +690,10 @@ __declspec(naked) void calculate_model_lod_detour()
 	}
 }
 
-typedef bool(__cdecl* fn_c000bd114_t)(e_skulls);
+typedef bool(__cdecl* fn_c000bd114_t)(e_skull_type);
 fn_c000bd114_t p_fn_c000bd114;
 
-bool __cdecl fn_c000bd114_IsSkullEnabled(e_skulls skull_index)
+bool __cdecl fn_c000bd114_IsSkullEnabled(e_skull_type skull_index)
```

Comment:
might as well rename this function while at it, e.g. `is_skull_enabled_hook()`

Reply:
Nice catch, although for the wrong reason.
We shouldn't even have this function in the first place since it's a duplicate detour